src/Tribe/Views/V2/Assets.php

```diff
@@ -99,6 +99,49 @@ public function register() {
 			]
 		);
 
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
+		// 	$plugin,
+		// 	'tribe-events-widgets-v2-skeleton',
+		// 	'widgets-skeleton.css',
```

Comment:
:no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 33c95dc71e4925259b2f3ab0e7206990</sub>